utility/response.js

```diff
@@ -0,0 +1,13 @@
+const _ = require('lodash');
+
+exports.responseBody = (text) => {
```

Comment:
`responseBody` sounds like a variable not a function. Specify what it does.

`generateResponseBody` or `buildResponseBody`